src/renderer/components/table/table-cell.tsx

```diff
@@ -15,6 +15,7 @@ export interface TableCellProps extends React.DOMAttributes<HTMLDivElement> {
   isChecked?: boolean; // mark checkbox as checked or not
   renderBoolean?: boolean; // show "true" or "false" for all of the children elements are "typeof boolean"
   sortBy?: TableSortBy; // column name, must be same as key in sortable object <Table sortable={}/>
+  showWithColumn?: string // className of another column, if it is not empty the current column is not shown in the filter menu, visibility of this one is the same as a specified column, applicable to headers only
```

Comment:
This is an optional property so I don't see how "some columns shouldn't be hidden" applies. 

I would say that this actually makes the action menu simpler for the users as we don't have to come up with some name if we want to hide a title-less column.

Reply:
@aleksfront Partially agreed with you, but how about the "icon" header in Apps->Charts, IMHO the column is just a visualization of the "Name" column and it should be shown as part of the name or the user should have the ability to manage this. 
Also, sometimes the rendering table looks glitchy when there is the first column is title-less and has no data:
![image](https://user-images.githubusercontent.com/53330707/102061556-0be53600-3e0d-11eb-9aa0-a6c032c60ee6.png)
(I agree with the fact that this set of columns has no sense, just to highlight the issue)
Probably we could hide the column (or set width = 0) in cases when this one has no shown objects and do not check it in showColumn(), from my pov it will be much better instead of showing empty column without possibility to hide it

Reply:
Regarding the checkbox as the title, is it possible to force it being empty by setting its title to `" "`? Or maybe even `{ title: "Selected", className: "hidden", ... }`?